Add warning about jsDOM

You should warn people about jsDOM. In many cases what is supported by jsDOM are not supported by browsers. Meaning server side tests with jsDOM can all be green, but soon as you test them in a browser environment, they fail.

Too many examples to write them all here, but I mentioned this because I once again ran into this issue:

```
AssertionError: expected '<div style="transition: none 0s ease 0s ;"></div>' to deeply equal '<div></div>'AssertionError@http://xxx/node_modules/chai/chai.js:5146:18
```

It works just fine on server side with jsDOM, but not browser side.

I mention this because of this ticket https://github.com/babel/generator-babel-boilerplate/issues/234

--- a/app/templates/test/setup/node.js
+++ b/app/templates/test/setup/node.js
@@ -9,6 +9,11 @@ require('./setup')();
 	Uncomment the following if your library uses features of the DOM,
 	for example if writing a jQuery extension, and
 	add 'simple-jsdom' to the `devDependencies` of your package.json
+
+  Note that JSDom doesn't implement the entire DOM API. If you're using
+  more advanced or experimental features, you may need to switch to
+  PhantomJS. Setting that up is currently outside of the scope of this
+  boilerplate.
 */
 // import simpleJSDom from 'simple-jsdom';
 // simpleJSDom.install();

